# The SICK Laser discription file of the robot in habitat
sick_laser:

  type: "2D_Laser"
  name: "sick_laser"
  frame: "/laser"
  topic: "laser"
  laserscan_topic: "/scan"
  publish_transfromation: True
  position: [0.0, 0.0, 0.1]

  sensor_info:

    far: 100.0
    near: 0.25

    hfov: 360

    accuracy: 0.03
    rate: 20

    resolution:
      horizontal: 512
      vertical: 512 # must be the same to horizontal resolution to force habitat rendering the horizontal as the expected resolution.